contracts/contracts/utils/InitializableAbstractStrategy.sol

```diff
@@ -206,6 +302,37 @@ abstract contract InitializableAbstractStrategy is Initializable, Governable {
         IERC20(_asset).safeTransfer(governor(), _amount);
     }
 
+    /**
+     * @dev Set the reward token addresses.
+     * @param _harvesterAddress Address of the harvester
+     */
+    function setHarvesterAddress(address _harvesterAddress)
+        external
+        onlyGovernor
+    {
+        harvesterAddress = _harvesterAddress;
+        emit HarvesterAddressesUpdated(harvesterAddress, _harvesterAddress);
+    }
+
+    /**
+     * @dev Set a fee in basis points to be rewarded when calling harvest
+     * @param _harvestRewardBps Basis point fee to be rewarded
+     */
+    function setHarvestRewardBps(uint32 _harvestRewardBps) external onlyGovernor {
+        require(_harvestRewardBps <= 1000, "Harvest reward fee should not be over 10%");
```

Comment:
@DanielVF you fine with having a 10% hard limit here as well?

Reply:
Yes.